src/main.cpp

```diff
@@ -207,6 +207,7 @@ static void lPrintVersion() {
     printf("    [--print-target]\t\t\tPrint target's information\n");
     printf("    [--fuzz-test]\t\t\tRandomly perturb program input to test error conditions\n");
     printf("    [--fuzz-seed=<value>]\t\tSeed value for RNG for fuzz testing\n");
+    printf("    [--[no-]discard-value-names]\t\tDo not discard/Discard value names when generating LLVM IR.\n");
```

Comment:
An option with [no-] should be sorted in alphabetical order as if [no-] is not present.
Could you sort the whole output for `devUsage()`? The sorting broken currently.

Reply:
Done